json-fleece-openapi3/src/Fleece/OpenApi3.hs

```diff
@@ -290,7 +290,11 @@ lookupResponse operationKey schemaMap responseRef =
                   Just OA.OpenApiBoolean ->
                     pure . Just $ CGU.boolSchemaTypeInfo
                   Just OA.OpenApiInteger ->
-                    pure . Just $ CGU.integerSchemaTypeInfo
+                    case OA._schemaFormat schema of
+                      Just "int32" -> pure . Just $ CGU.int32SchemaTypeInfo
+                      Just "int64" -> pure . Just $ CGU.int64SchemaTypeInfo
+                      Just format -> responseError $ "Unknown integer schema format " <> T.unpack format <> "."
```

Comment:
In the other cases in the codgen we treat unrecognized integer formats as the same as not specifying one rather than erroring out. I think it makes sense to do the same here.

Reply:
Changed in 80f28125479bfa68bbec1b1a15ee826a5cff3de1 to use the integer schema instead of `Nothing`.